src/EffectTransform.js

```diff
@@ -133,15 +133,19 @@ class EffectTransform {
         const uniforms = drawable.getUniforms();
         const effects = drawable.getEnabledEffects() & effectMask;
 
-        if ((effects & ShaderManager.EFFECT_INFO.ghost.mask) !== 0) {
-            // gl_FragColor.a *= u_ghost
-            dst[3] *= uniforms.u_ghost;
-        }
-
         const enableColor = (effects & ShaderManager.EFFECT_INFO.color.mask) !== 0;
         const enableBrightness = (effects & ShaderManager.EFFECT_INFO.brightness.mask) !== 0;
 
         if (enableColor || enableBrightness) {
```

Comment:
Can you comment here to say that what we're doing is removing the premultiplied alpha because color and brightness effects need to be applied to non-premultiplied-alpha colors?

Reply:
:+1: